### ISSUE TITLE
[Not an issue, just a question] Nexus site management

### ISSUE THREAD

**Issue by GrantSP** [TanninOne/modorganizer#499]
@Tannin do you think you could do a bit of house-cleaning on the Nexus pages?
There are still users posting questions on the Fallout New Vegas site, that page should probably have its comments removed or at least blocked.

Also the main Skyrim page has the 'Forums' link active and a lot of users are posting questions there. Didn't even realise it was active until I went looking for a post. All those questions should be asked either here, STEP or on the main 'Posts' tab. I don't know how many questions have gone unanswered by those users.

Basically there are too many places that users are posting questions and not getting responses.

Just a suggestion!

**Comment by TanninOne** [comment]
I had a sticky on each of the "old" games asking people to head to the skyrim page. It seems they got unstickied at some point, so I now fixed it.

The Skyrim discussion forum has a sticky in large red letters telling people where to ask for help. I get tons of emails and PMs as well. And people post in separate nexus threads, bethsoft forum, certain adult-mod forums, non-english forums, reddit, ...
I can't stop people from posting in other channels, what's the point in suppressing a few?
I tell people where they should post, where they have the best chance to get a proper answer, the rest is up to them.
It's your choice to read or not read the nexus comment section and answer there or not.
I want to write a software not manage communities.

**Comment by GrantSP** [comment]
Please don't view this as presumptuous but you could assign someone the task of managing those pages? The comments sections can be disabled so users CAN'T post there.
I understand you have no control of users in other fora like Reddit, but you do have control of Nexus, STEP and here.
I know you want to focus on writing software, but you have created these communities and thus have a measure of control on what is presented.
All I want to do is continue to highlight MO as the pre-eminent mod management tool around.
